tests/integ/modin/index/test_argmax_argmin.py

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (c) 2012-2024 Snowflake Computing Inc. All rights reserved.
+#
+
+import modin.pandas as pd
+import pandas as native_pd
+import pytest
+
+import snowflake.snowpark.modin.plugin  # noqa: F401
```

Comment:
If this is imported but unused, can it be removed?  Otherwise if it's used then don't need the flake8 annotation.

Reply:
This is how we've been importing the modin plugin, flake8 raises an error if we don't do it this way. But it is used.